params.go

```diff
@@ -124,6 +124,11 @@ type Params struct {
 	// Account is deprecated form of StripeAccount that will do the same thing.
 	// Please use StripeAccount instead.
 	Account string
+
+    // StripeVersion sets the version for the API to use when parsing requests
+    // and rendering responses.  This string refers to a date, such as
+    // "2017-05-25"
+    StripeVersion string
```

Comment:
OK, I added a general `Headers` field here instead, and `StripeVersion` is now on `EphemeralKeyParams`.